src/plotter_disk.hpp

```diff
@@ -73,12 +73,11 @@ class DiskPlotter {
         bool show_progress = false)
     {
         // Increases the open file limit, we will open a lot of files.
-#ifndef _WIN32
```

Comment:
you need to put these `#ifndef` back. Note the `n` in there, this condition is to omit this code on windows, because windows doesn't have rlimits. (this is why this patch fails to build on windows, [here](https://github.com/Chia-Network/chiapos/pull/202/checks?check_run_id=2233385499#step:8:235))